### DISCUSSION TITLE
Profile Page is not loading

### DISCUSSION THREAD

**Discussion by Saptarshi-Chakraborty** [community/community#130993] [Mobile]
Today I am trying to open my GitHub profile in the Android app, but it is not opening. The page is showing try again button. All the other pages in the app is working, but not the profile page.

**Comment by Claudioleroi** [comment]
Step-by-Step Troubleshooting Guide
Check Your Internet Connection:

Ensure that your internet connection is stable. Try switching between Wi-Fi and mobile data.
Restart the App:

Close the GitHub app completely (remove it from recent apps) and then reopen it.
Clear Cache and Data:

Go to your device's settings.
Navigate to Apps or Application Manager.
Find and tap on the GitHub app.
Tap on Storage.
Clear Cache and then Clear Data. Note that clearing data will log you out, so make sure you have your login credentials.
Update the App:

Go to the Google Play Store.
Search for the GitHub app.
If an update is available, tap Update.
Reinstall the App:

Uninstall the GitHub app from your device.
Go to the Google Play Store and reinstall the GitHub app.
Check GitHub Status:

Sometimes the issue might be with GitHub's servers. Check the status of GitHub's services at [GitHub Status](https://www.githubstatus.com/).
Check for Device Issues:

Restart your Android device.
Ensure your device's software is up to date.
Try a Different Device or Browser:

Try accessing your GitHub profile from a different device or through a web browser to see if the issue persists.
If the Issue Persists
If none of the above steps resolve the issue, it might be a bug in the app. In that case, you can report the issue to GitHub support:

Open the GitHub App:

If the profile page still does not work, navigate to another section of the app that does.
Go to Settings or Help Section:

Look for a Help or Support section in the app settings.
Report the Issue:

Use the app's feedback or support feature to report the issue. Provide as much detail as possible, including screenshots if possible.

**Comment by Digitix08** [comment]
That page, like **never** worked for me. Since the moment i installed the GitHub app (from which i am writing this), only 3 of the 4 pages worked. I recommend the devs remove that page since they did not fix it in almost a year

**Reply by Saptarshi-Chakraborty** [reply]
Soo true
I uninstalled the app and reinstalled it. Still that single page is not working

**Reply by ByteToHex** [reply]
Having the same experience and just happened upon this thread.

**Comment by iskae** [comment]
Hey folks 👋 

Thank you so much for reaching out and I am sorry for causing trouble for you all.

I would love to understand what is going wrong with it on our side and would appreciate your help 🙇 

- If you also have an iOS device, are you able to see your profile on iOS?
- Do you receive any error messages or any request ids that we can follow through?
- If it also happens when viewing someone else's profile, can you please let us know the handle of the user?

Thanks!

**Comment by Arcdashckr** [comment]
After clearing the cache and data, I logged in again and it worked. 17.04.2026

**Comment by dorukcodes** [comment]
I had something similar happen before, it was usually a caching issue on the app. Try clearing the app cache or logging out and back in. Updating or reinstalling the app can also fix it. If it still doesn’t work, it’s probably a temporary app bug on GitHub’s side.

**Comment by Digitix08** [comment]
Found the fix. In my case, it was that i changed my username on github while logged in to github mobile. The app continued working and showing me as logged in to my new username, but the profile page tried to show the page of my old username, which does no longer exist
